services/news/src/index.ts

```diff
@@ -53,6 +56,26 @@ app.get('/', async (req: Request, res: Response) => {
   });
 });
 
+app.get('/uc-publisher-ads-req', async (req: Request, res: Response) => {
+  const {auctionType} = req.query;
+  const bucket = req.query.key;
+  const cloudEnv = req.query.env;
+
+  res.render('uc-publisher-ads-req', {
+    title: PUBLISHER_ADS_REQ_TITLE,
+    lorem: TEXT_LOREM,
+    EXTERNAL_PORT,
+    HOME_HOST,
+    DSP_HOST,
```

Comment:
Remove DSP_HOST. See detailed comment in EJS template.

Reply:
Done.